packages/flutter/lib/src/rendering/flex.dart

```diff
@@ -152,7 +155,11 @@ class RenderFlex extends RenderBox with ContainerRenderObjectMixin<RenderBox, Fl
     }
   }
 
-  /// The limit used to compute free space along the main axis.
+  /// How much space space should be occupied in the main axis.
+  ///
+  /// After allocating space to children, there might be some remaining free
+  /// space. This value controls whether to maximize or minimize the amount of
+  /// free space, subject to the incoming layout contraints.
```

Comment:
maybe mention that a tight flexible child is going to make this get essentially ignored?